app/utils.py

```diff
@@ -24,23 +37,41 @@ def cosine_similarity(peer_v, query_v):
     den_b = dot(query_v, query_v)
     return num / (sqrt(den_a) * sqrt(den_b))
 
+#################################################
+# Load entropy list
+#################################################
+
+def loadEntropies():
+	entropies_dict={}
+	entropies_file = os.path.join(
+	    os.path.dirname(__file__),
+	    "ukwac.entropy.txt")
```

Comment:
@minimalparts Have a look at this: https://github.com/minimalparts/PeARS/pull/41/files
Please do merge it if this fix is working for you as well.